custom/enikshay/integrations/ninetyninedots/views.py

```diff
@@ -52,117 +38,6 @@ class UnenrollPatientRepeaterView(AddCaseRepeaterView):
-    except NinetyNineDotsException as e:
-        notify_exception(
-            request,
-            message=("An error occurred updating the episode case after receiving a 99DOTS"
-                     "adherence case for beneficiary {}. {}").format(beneficiary_id, e))
-    return json_response({"success": "Patient adherences updated."})
-
-
-@toggles.NINETYNINE_DOTS.required_decorator()
-@api_auth
-@two_factor_exempt
-@require_POST
-@csrf_exempt
-@check_domain_migration
-def update_patient_details(request, domain):
-    try:
-        request_json = json.loads(request.body)
-    except ValueError:
-        return json_response({"error": "Malformed JSON"}, status_code=400)
-
-    try:
-        PatientDetailsUpdater(domain, request_json).update_cases()
-    except NinetyNineDotsException as e:
-        return json_response({"error": six.text_type(e)}, status_code=400)
-
-    return json_response({"success": "Patient details updated."})
-
-
-@toggles.NINETYNINE_DOTS.required_decorator()
-@api_auth
-@two_factor_exempt
-@require_POST
-@csrf_exempt
-@check_domain_migration
-def update_adherence_confidence(request, domain):
-    try:
-        request_json = json.loads(request.body)
-    except ValueError:
-        return json_response({"error": "Malformed JSON"}, status_code=400)
-    beneficiary_id = request_json.get('beneficiary_id')
-    start_date = request_json.get('start_date')
-    end_date = request_json.get('end_date')
-    confidence_level = request_json.get('confidence_level')
-
-    try:
-        validate_beneficiary_id(beneficiary_id)
-        validate_dates(start_date, end_date)
-        validate_confidence_level(confidence_level)
-        update_adherence_confidence_level(
-            domain=domain,
-            person_id=beneficiary_id,
-            start_date=parse_datetime(start_date),
-            end_date=parse_datetime(end_date),
-            new_confidence=confidence_level
-        )
-    except NinetyNineDotsException as e:
-        return json_response({"error": e.message}, status_code=400)
-
-    return json_response({"success": "Patient adherences updated."})
-
-
-@toggles.NINETYNINE_DOTS.required_decorator()
-@api_auth
-@two_factor_exempt
-@require_POST
-@csrf_exempt
-@check_domain_migration
-def update_default_confidence(request, domain):
-    try:
-        request_json = json.loads(request.body)
-    except ValueError:
-        return json_response({"error": "Malformed JSON"}, status_code=400)
-
-    beneficiary_id = request_json.get('beneficiary_id')
-    confidence_level = request_json.get('confidence_level')
-
-    try:
-        validate_beneficiary_id(beneficiary_id)
-        validate_confidence_level(confidence_level)
-        update_default_confidence_level(domain, beneficiary_id, confidence_level)
-    except NinetyNineDotsException as e:
-        return json_response({"error": e.message}, status_code=400)
-
-    return json_response({"success": "Default Confidence Updated"})
```

Comment:
you can delete all the functions below this too

Reply:
Ya.. there were some tests related to it.. didn't want to get too much in depth of this.. and just remove the functionalities. Leaving those utility methods since they are harmless.